src/eventhub.ts

```diff
@@ -19,6 +19,7 @@ THE SOFTWARE.
 */
 
 import WebSocket from 'isomorphic-ws';
+import events from "events";
```

Comment:
Please keep consistent quotes. And import the new way - not the old compat way.
```suggestion
import EventEmitter from 'events';
```
https://github.com/browserify/events/blob/master/events.js#L60